### ISSUE TITLE
Lighthouse Data Studio connector shows the latest score on hover modal

### ISSUE THREAD

**Issue by adamhenson** [foo-software/bugs#11]
# Summary

This could be missing configuration in the Data Studio template, otherwise if its not available, we'll need to workaround by updating the connector code.

# Actual Behavior

On hover state, the **latest** score for the day is displayed.

# Expected Behavior

On hover state, the **average** score for the day is displayed.

**Comment by adamhenson** [comment]
It seems that this is more of an issue with the template than the connector itself. It's not completely clear yet, but it seems that you can achieve the ideal state by adjusting the following in the report.

- Toggle the report in edit mode.
- Select the chart. Edit each metric to use `Running average` as the `Running calculation`.
- Select each score card and edit each to also use `Running average`.

<img width="405" alt="Screen Shot 2020-12-21 at 10 39 17 PM" src="https://user-images.githubusercontent.com/747743/102852474-57d74280-43ec-11eb-8003-79add2e21e4a.png">

From what I can tell, by setting the above - each day of the chart will be an average of that day. The score cards will be an average of all scores in the timeline (not necessarily the average of all days). Will do some more testing to confirm.

**Comment by adamhenson** [comment]
Documentation around Data Studio is a bit blurry. Currently we're pushing every lighthouse audit to a row, regardless of the date being the same (must format like `YEAR_MONTH_DAY`). Intuitively one would expect Data Studio to average rows for a single day or provide this option... and it seems that we misunderstood this as being the default behavior. Also, `Running average` doesn't seem to provide this behavior.

**tl;dr** - seems like we need to calculate daily averages in the connector code and resubmit to Google. Approval process typically takes a couple weeks to a couple months. Will post again when we've resubmitted.

**Comment by adamhenson** [comment]
Connector updates have been made and tested. Submitted to Google for review. Upon submission - Google says 2-3 weeks to review, but I've experienced a lot of back and fourth in the past.

**Comment by adamhenson** [comment]
Turns out, we didn't need to resubmit to make updates to the connector. These updates are now published and the issue is fixed.
